packages/flutter/lib/src/material/text_theme.dart

```diff
@@ -355,12 +355,20 @@ class TextTheme with Diagnosticable {
   /// [displaySmall], [headlineLarge], [headlineMedium], and [bodySmall]. The
   /// `bodyColor` is applied to the remaining text styles.
   ///
+  /// The `displayFontFamily` is applied to [displayLarge], [displayMedium],
+  /// [displaySmall], [headlineLarge], [headlineMedium], and [bodySmall]. The
```

Comment:
Why would display apply to bodySmall?

Reply:
And how does it relate to the headline ones as well?

Reply:
Hey this was done basis what `displayColor` follows. Based on the reply I received here: https://discord.com/channels/608014603317936148/608021234516754444/1242440562452332565 I assume this is not relevant now?